charts/Controls.tsx

```diff
@@ -599,7 +599,7 @@ export class Controls {
     @computed get hasInlineControls(): boolean {
         const { chart } = this.props
         return (
-            chart.tab === "chart" &&
+            (chart.tab === "chart" || chart.tab === "table") &&
```

Comment:
What about the idea of having fewer conditionals and passing in controls?

Reply:
I tried it, but realized the same conditionals would just move up into ChartView, which isn't necessarily a better place for them to be.